### DISCUSSION TITLE
Android SDK When?

### DISCUSSION THREAD

**Discussion by DKB0512** [supabase/supabase#1646] [Questions] [answered]
I just want to know that when are you going to release Android SDK with Java & Kotlin support?
Is it planned in the road-map?

**Comment by awalias** [comment, marked as answer]
hey @DKB0512 one of our community members contributed 2 libs so far for Kotlin:

https://github.com/supabase/postgrest-kt
https://github.com/supabase/gotrue-kt

we're always looking for community maintainers and contributors to extend these offerings, it probably won't be completed internally within the next 3 months
